code/index.html

```diff
@@ -1,30 +1,79 @@
 <!DOCTYPE html>
 <html>
-  <head>
+
+<head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
-  </head>
-  <body>
-    <!-- video or image as a header is cool :) -->
-
-    <!-- Signup form -->
-
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
-
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
-      </div>
-
-      <div class="question" id="section2">Section 2</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
-      </div>
-    </section>
-    <!-- dont forget to link the javascript file here! -->
-  </body>
-</html>
+    <title>Australia</title>
+    <link rel="stylesheet" href="./style.css">
+</head>
+
+<body>
+
+    <!--HERO SECTION CONSIST OF TITLE AND IMAGES/VIDEOS-->
+    <class="hero">
+        <header>
+            <picture>
+                <source srcset="./videos/imageone.jpg" media="(max-width: 767px)">
+                <source srcset="./videos/imagetwo.jpg" media="(max-width: 1199px)">
+                <img class="hero-image" src="./videos/imagethree.jpg" alt="Heroimage">
+            </picture>
+            <h1 class="hero-title"> TRIP TO AUSTRALIA </h1>
+
+            <!--if you want to overlapp use position relative relation. 
+    To define its parent class to position relative  and to place the element in absolute!!!-->
+        </header>
+
+        </class>
+
+        <!-- Signup form -->
+        <div class="accordion">
+
+            <h3 id="accordion-header-1" class="accordion-header">Do I need to get tested before I travel?</h3>
+            <p id="accordion-content-1" class="accordion-content">Yes</p>
+        </div>
+
+        <div class="accordion">
+            <h3 id="accordion-header-2" class="accordion-header">How much does it cost to rent a car per day?</h3>
+            <p id="accordion-content-2" class="accordion-content">AUD 100</p>
+
+        </div>
+
+        <div class="accordion">
+            <h3 id="accordion-header-3" class="accordion-header">How much is Tax on air travel?</h3>
+            <p id="accordion-content-3" class="accordion-content">Please email us for a price quote.</p>
+
+        </div>
+
+
+        <form class="form">
+            <ul class="form_list">
+                <li class="form_list-item">
+                    <input type="text" name="first_name" required placeholder="First Name" class="form_input" />
+                    <label for="first_name" class="form_label">First Name</label>
```

Comment:
You need an `id` attribute (that is the same as the `for` attribute on the label) on the input for them to connect. Read more in the general feedback document

Reply:
Possibly, you could also consider omitting the `ul` and `li` elements and style the labels and inputs directly. Maybe having the labels before the inputs for a more intuitive design on mobile. It's reversed now:
<img width="205" alt="Skärmavbild 2021-02-04 kl  17 41 08" src="https://user-images.githubusercontent.com/51529201/106925224-2c948280-6710-11eb-944c-0ee2b92bfdf1.png">